src/Microsoft.DotNet.Cli.Utils/CsvFormatter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal static class CsvFormatter
+    {
+        internal static string EscapeRow(IEnumerable<string> values)
+        {
+            return values
+                .Select(EscapeValue)
+                .Aggregate((a, v) => a + "," + v);
+        }
+
+        internal static string EscapeValue(string value)
+        {
+            return "\"" + value.Replace("\\", "\\\\").Replace("\"", "\\\"") + "\"";
```

Comment:
It's totally not designed to be perfectly adhering to the "spec" (which is a kinda weird spec since it was created **after** a number of various formats were in use ;)). We will be dramatically reviewing the deps file though and likely moving away from CSV anyway so I agree we don't need to worry about this until we've resolved that issue. #394